menu.js

```diff
@@ -0,0 +1,66 @@
+// #2
+function showMenu() {
+
+  const min = 1;
+  const max = 100;
+
+  // function getRandomInt(min, max) {
+  //   return Math.floor(Math.random() * (max - min + 1)) + min;
+
+
+  const menu = {
+    1 : {
+      id: Math.floor(Math.random() * (max - min + 1)) + min,
```

Comment:
It would make more sense to use the same id as was used for the key here. Otherwise we end up with inconsistent ids on every executing. Also, by using random we run the risk of duplicate ids which may cause strange bugs